changelog.d/pa-1910.fixed

```diff
@@ -0,0 +1 @@
+Added a fix for a bug involving parsing of TS imports.
```

Comment:
It's very difficult to understand what was the bug about just reading this. The changelog entries should be a bit more detailed. This has been a common complain from @adamkvitek when trying to produce release notes.